connectors/kafka/element-templates/kafka-inbound-connector-intermediate.json

```diff
@@ -0,0 +1,266 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Kafka Consumer Connector",
+  "id": "io.camunda.connectors.inbound.KafkaIntermediate.v1",
+  "version": 1,
+  "description": "Consume Kafka Message",
+  "documentationRef": "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/kafka/",
```

Comment:
Just FYI: did a proposal to restructure docs. [Here](https://github.com/camunda/camunda-platform-docs/pull/2017#issuecomment-1538112618).